Brain_I2C/Brain_I2C_Basic.ino

```diff
@@ -0,0 +1,26 @@
+#include <Wire.h>
+
+void setup() {
+  Wire.begin();        // join i2c bus (address optional for master)
+  Serial.begin(9600);  // start serial for output
+}
+
+void loop() {
+  int sensor_id = 0; //change to desired sensor_id
+  Wire.requestFrom(8, 4); // request 4 bytes from address 8
```

Comment:
Instead of setting a specific sensor id in the loop, we want to ahve a global map of sensor id to I2c address and loop through each of those and do wire.request() and get data and stuff.

Reply:
So should I have a globally defined dictionary that maps a sensor to a tuple (address of board, sensor ID on that board)? And then in the loop, just go through each of those sensors?